app/screens/localization/LanguageSelection.tsx

```diff
@@ -119,15 +120,15 @@ const LanguageSelection = ({ route, navigation }: Props): any => {
         } else {
 
           if (route.params != undefined) {
-            newCountryId = route.params.country.countryId;
+            newCountryId = route.params.country?.countryId;

```

Comment:
Can params be null? If yes should we add "?"

Reply:
added the ? for handle if params getting null